src/layouts/default.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import { Box } from '../components/publiq-ui/Box';
+import { Inline } from '../components/publiq-ui/Inline';
+import { SideBar } from '../components/SideBar';
+
+const Default = ({ children }) => {
+  return (
+    <Inline>
+      <SideBar />
+      <Box
+        css={`
+          flex: 1;
+          height: 100vh;
+          overflow-x: hidden;
+          overflow-y: auto;
+        `}
+      >
+        {children}
+      </Box>
+    </Inline>
+  );
+};
+
+Default.propTypes = {
+  children: PropTypes.node,
+};
+
+export default Default;
```

Comment:
I think this makes sense to name default in Nuxt, because Nuxt automatically loads it as the default template.

But, it looks really weird when you use the component yourself. Could we give it a more descriptive name?